core/domain/collection_services.py

```diff
@@ -278,31 +340,48 @@ def get_collection_titles_and_categories(collection_ids):
 
 def get_completed_exploration_ids(user_id, collection_id):
     """Returns a list of explorations the user has completed within the context
-    of the provided collection. Returns an empty list if the user has not yet
-    completed any explorations within the collection. Note that this function
-    will also return an empty list if either the collection and/or user do not
-    exist.
-
-    A progress model isn't added until the first exploration of a collection is
-    completed, so, if a model is missing, there isn't enough information to
-    infer whether that means the collection doesn't exist, the user doesn't
-    exist, or if they just haven't mdae any progress in that collection yet.
-    Thus, we just assume the user and collection exist for the sake of this
-    call, so it returns an empty list, indicating that no progress has yet been
-    made.
+    of the provided collection.
+
+    Args:
+        user_id: str. ID of the given user.
+        collection_id: str. ID of the collection.
+
+    Returns:
+        list(Exploration). A list of explorations that the user with the given
+        user id has completed within the context of the provided collection with
+        the given collection id. The list is empty if the user has not yet
+        completed any explorations within the collection, or if either the
+        collection and/or user do not exist.
+
+        A progress model isn't added until the first exploration of a collection
+        is completed, so, if a model is missing, there isn't enough information
+        to infer whether that means the collection doesn't exist, the user
+        doesn't exist, or if they just haven't mdae any progress in that
+        collection yet. Thus, we just assume the user and collection exist for
+        the sake of this call, so it returns an empty list, indicating that no
+        progress has yet been made.
     """
     progress_model = user_models.CollectionProgressModel.get(
         user_id, collection_id)
     return progress_model.completed_explorations if progress_model else []
 
 
-def get_valid_completed_exploration_ids(user_id, collection_id, collection):
+def get_valid_completed_exploration_ids(user_id, collection):
     """Returns a filtered version of the return value of
     get_completed_exploration_ids, where explorations not also found within the
     collection are removed from the returned list.
+
+    Args:
+        user_id: str. ID of the given user.
+        collection: Collection. Collection of the given collection id.
```

Comment:
There's no "given collection id", but I think you can just strike "Collection of the ..." onwards since it's pretty clear from context what this is.